benchmark/runner/conf/default.yaml

```diff
@@ -1,8 +1,9 @@
 iterations: 12
 threads: null
+timeout: 1200
 savina_path: "${oc.env:SAVINA_PATH}"
 lf_path: "${oc.env:LF_PATH}"
-continue_on_error: False
+continue_on_error: True
```

Comment:
I think @petervdonovan might have a better idea about this.

Reply:
I did this a while ago because it was not possible to run the benchmarks without it. Since the `PEDF_NP` scheduler is not currently in use, I think we can revert this change.